src/gamepad/mapping-error.vala

```diff
@@ -0,0 +1,5 @@
+// This file is part of GNOME Games. License: GPLv3
+
+public errordomain LibGamepad.MappingError {
+	NOT_A_MAPPING
```

Comment:
Let's add commas after all the error codes and enumerated values (this is another implicit rule that need to be added to CodingStyle.txt).

It is better because when changing the error codes we don't change unrelated lines by adding or removing a comma, making managing the code in the version control system easier.